### PR TITLE
Added iOS 8 support.

### PR DESCRIPTION
**Pull request by PrajeetShrestha**
Added code to support ios-arkit for iOS 8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,13 @@ At this moment you should be enjoying a fully featured augmented reality view sh
 
 And that's all!! For more information on how to add custom overlay views, configure the radar position, start the engine in different orientations or better understanding of the front/floor looking feature, please run the provided example project.
 
+## For iOS 8 Support
+Add these key in Info.plist of your app
```
Comment (calonso) [inline README.md]: keys, in plural :)

File: README.md [hunk 2]
```diff
@@ -70,6 +70,13 @@ At this moment you should be enjoying a fully featured augmented reality view sh
...
+NSLocationAlwaysUsageDescription
```
Comment (calonso) [inline README.md]: Maybe add some more info on what the key means, and which value is expected. Or a link to the documentation?

**Comment by calonso** [comment]
Thanks @PrajeetShrestha, just a couple of minor comments left there before merging.

**Comment by PrajeetShrestha** [comment]
Ok added the descriptions to the keys.

**Comment by calonso** [comment]
@PrajeetShrestha That's great. Thank you very much for the PR!

**Comment by PrajeetShrestha** [comment]
It's a great library! Thank you very much. :) Seems like the hound is taking too much time reviewing changes :)

**Comment by calonso** [comment]
Great!! Any other improvements? Suggestions?

I had in mind to publish it as a cocoapod. What do you think?

**Comment by PrajeetShrestha** [comment]
It would be too handy if you make Pod of it. I am exploring your library in details starting from today since I have to implement it in a project. I will be definitely sending you some more pull requests / comments if I find a room for great improvement in it.

**Comment by calonso** [comment]
Perfect, let me know if you need a hand.

**Pull request merged**